test/project-config/forms/contact/c50_supervision_area-edit.properties.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Do we need contact forms (add and edit)?

Reply:
Good question.
Are they not needed while configuring the levels of stock monitoring ?
If so then we can just keep the one needed and get rid of the rest.